lib/Mojolicious/Plugin/DefaultHelpers.pm

```diff
@@ -42,6 +42,14 @@ sub register {
   $app->helper(dumper  => sub { shift; dumper @_ });
   $app->helper(include => sub { shift->render_to_string(@_) });
 
+  $app->helper(
+    log => sub {
+      my $c = shift;
+      return $c->stash->{'mojo.log'}
+        ||= $c->app->log->context('[' . $c->req->request_id . ']');
+    }
+  );
```

Comment:
Does this mean that if I want my log messages to have a different context formatting, then I need to redefine the whole helper? (As in, if I want to remove the brackets or something) Could we change the log helper to take a $log object to set the value of `mojo.log`, or will that make the helper too slow?
(Not a big issue to me btw, since I think the default makes sense)

Reply:
Yes, just redefined the helper, that's what helpers are for after all. For most people the default will be just fine anyway.

Reply:
Or even add more log helpers with different contexts.